chapter_15_advanced_forms.asciidoc

```diff
@@ -141,6 +141,17 @@ it can be hard to see what's being tested:
 AssertionError: ValidationError not raised
 ----
 
+// SEBASTIAN: for scenarios that are meant to test if something IS NOT raising exception, I use:
+// ```
+// def test_():
+//     ...
+//     try:
+//         model.full_clean()
+//     except ValidationError:
+//         self.fail("Should not raise exception!")
+// ```
+// SEBASTIAN continued: WDYT?
```

Comment:
@Enforcer nice!  you'd. get a more explicit error message in the test fail this way.  otoh it is a bit more verbose...